luda/editor/client/src/timeline/cameraTrack/CameraTrackBody.ts

```diff
@@ -110,6 +110,8 @@ function pushClipsForward(state: TimelineState, clips: Clip[]) {
     const { clipId } = state.actionState;
     const draggingClip = clips.find((clip) => clip.id === clipId);
     if (!draggingClip) {
+      // TODO: It may be a clip of another track.
+      return;
       throw new Error("clip not found");
     }
```

Comment:
If dragging clip is other track's clip. It will be undefined

Reply:
Also here
https://github.com/NamseEnt/namseent/pull/18/files#diff-9afdf27459f954adc859a72610390222c897bd4840f471599a53ea8c1157bad3R174-R175